portfolio/src/main/webapp/script.js

```diff
@@ -32,17 +32,30 @@ function addRandomFact() {
  * Fetch the list of comments and add them to the page
  */
 function fetchAndAddComments(){
-  fetch('/comments').then(response => response.json()).then((comments) => {
+  fetch('/comments').then(response => response.json()).then((comment) => {
     const commentContainer = document.getElementById('comments-container');
-    comments.forEach(function(comment) {
-      commentContainer.appendChild(createListElement(comment))
+    comment.forEach(function(comment) {
+      commentContainer.appendChild(createCommentElement(comment))
```

Comment:
This is very confusing. "comment" is singular but you're invoking a forEach function on it. comment is a list, right? So this should be "comments", instead